packages/insomnia-app/app/ui/components/settings/general.js

```diff
@@ -508,6 +502,25 @@ class General extends React.PureComponent<Props, State> {
             as request data, names, etc.
           </p>
         </div>
+
+        {session.isLoggedIn() && (
+          <React.Fragment>
+            <hr />
+            {this.renderBooleanSetting('Enable version control beta', 'enableSyncBeta', '', true)}
+          </React.Fragment>
+        )}
+
+        {appConfig().appId === APP_ID_DESIGNER && (
```

Comment:
```suggestion
        {constants.getAppId() === APP_ID_DESIGNER && (
```

Favor consistency with other bits of code that use `getAppId()`